db/migrations/20170418171151_create_movies_locations.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+exports.up = function (Knex, Promise) {
+  return Knex.schema.createTable('movies_locations', (table) => {
+    table.increments('id').primary();
+    table.integer('loc_id');
```

Comment:
Nit: `location_id`. Its not a hard requirement (although it may be for some web frameworks/ORMs) but generally the foreign key should be named using the singular form of the foreign table name. `locations` => `location_id`.